package.json

```diff
@@ -2,7 +2,7 @@
     "name": "mindaro",
     "displayName": "Bridge to Kubernetes",
     "description": "Rapid Kubernetes development for teams",
-    "version": "2.0.120230716",
+    "version": "2.0.120230817",
```

Comment:
Package-lock has several changes due to rearranging, just leaving a comment FYI.